### ISSUE TITLE
Noticed a very small typo in the textbook

### ISSUE THREAD

**Issue by kevinchua6** [nus-cs2103-AY2122S1/forum#446]
![image](https://user-images.githubusercontent.com/45702380/142839717-440af5e3-d7c5-430e-bdab-241547fefb1e.png)

I noticed `notifyUIs` doesn't follow the Java coding standards (below) in [this section](https://nus-cs2103-ay2122s1.github.io/website/schedule/week11/topics.html#design-design-patterns-observer-pattern-what). Should probably be `notifyUis` instead

![image](https://user-images.githubusercontent.com/45702380/142840057-fdcf5e61-2035-4d99-b85f-96b3c417a631.png)

Sorry, I'm still in the pe mood and this has been bugging me :p

**Comment by kaixin-hc** [comment]
On the topic of minor typos, I also found some other minor typos 😅... just going to jump on this thread so as not to clutter the forum.

<img width="762" alt="Screenshot 2021-10-26 at 12 10 06 AM" src="https://user-images.githubusercontent.com/59471320/142861195-5c97c84f-e681-47dc-9806-c6a0afb563a2.png" >
---
<img width="972" alt="Screenshot 2021-09-28 at 10 05 11 PM" src="https://user-images.githubusercontent.com/59471320/142861202-81a0dddb-2944-4c79-b7e7-45192ebdbdbc.png" >
---
<img width="802" alt="Screenshot 2021-11-15 at 5 08 01 PM" src="https://user-images.githubusercontent.com/59471320/142861204-919af480-bcb8-414d-b480-1c6517552d73.png" >
---
<img width="1144" alt="Screenshot 2021-09-28 at 10 11 47 PM" src="https://user-images.githubusercontent.com/59471320/142861208-3a886754-4f5d-4096-b977-268be6a3cc49.png">
---
..

**Comment by fans2619** [comment]
I think the underline thing is intentional, not really underline, just using larger font to highlight

**Comment by damithc** [comment]
Thanks guys, keep them coming. Every bug report helps.

> I think the underline thing is intentional, not really underline, just using larger font to highlight

It's bug. MarkBind recently changed the syntax it uses for underlines. The previous syntax `++text++` is now used to making the text bigger. These are places where I overlooked when migrated to the new syntax.

**Comment by fans2619** [comment]
Oh I see!

**Comment by damithc** [comment]
> I noticed `notifyUIs` doesn't follow the Java coding standards (below) in [this section](https://nus-cs2103-ay2122s1.github.io/website/schedule/week11/topics.html#design-design-patterns-observer-pattern-what). Should probably be `notifyUis` instead

Good catch. I think I made an exception in that case because `notifyUis` is harder to read than `notifyUIs` (due to the additional lower case `s`).

**Comment by damithc** [comment]
> <img alt="Screenshot 2021-10-26 at 12 10 06 AM" width="762" src="https://user-images.githubusercontent.com/59471320/142861195-5c97c84f-e681-47dc-9806-c6a0afb563a2.png">


@kaixin-hc where did you find this? This was fixed sometime back in the module website. 


> ---
> <img alt="Screenshot 2021-09-28 at 10 05 11 PM" width="972" src="https://user-images.githubusercontent.com/59471320/142861202-81a0dddb-2944-4c79-b7e7-45192ebdbdbc.png">
> 

Good catch. Will fix.

> <img alt="Screenshot 2021-11-15 at 5 08 01 PM" width="802" src="https://user-images.githubusercontent.com/59471320/142861204-919af480-bcb8-414d-b480-1c6517552d73.png">
> 

Good catch. Will fix.

> <img alt="Screenshot 2021-09-28 at 10 11 47 PM" width="1144" src="https://user-images.githubusercontent.com/59471320/142861208-3a886754-4f5d-4096-b977-268be6a3cc49.png">
> 

Good catch. Will fix.

**Comment by kaixin-hc** [comment]
> @kaixin-hc where did you find this? This was fixed sometime back in the module website.
I took these screenshots over the course of the semester, and didn't think to check that they still worked with the website. I see that it is fixed on the module website now!

**Comment by damithc** [comment]
> I took these screenshots over the course of the semester

Nice. 👍 
Yes, it was fixed in the middle of the semester.

**Comment by trxe** [comment]
Adding to the thread some other small typos: 

![MSS](https://user-images.githubusercontent.com/23572486/142975881-2a742614-106d-4db1-9869-96bb26cedeaf.png)

I believe there should be a space here? i.e.  ". [" 

![Typo bug](https://user-images.githubusercontent.com/23572486/142975448-86048f53-9614-481f-a83e-d55029fd3a94.png)

**Comment by kaixin-hc** [comment]
<img width="685" alt="grammar" src="https://user-images.githubusercontent.com/59471320/143214919-d7c241c8-efad-4319-a720-0df9bc50bc92.png">
Grammatical issues where highlighted + I feel it would be clearer to put the quote in quotation marks, rather than in italics.

**Comment by AllardQuek** [comment]
Good catch @kaixin-hc! I don't think "tasks that needs" is a grammatical error though, because it is the **list** of tasks that needs the user's attention.

![image](https://user-images.githubusercontent.com/40263305/143260458-2ed77b82-0ad9-4242-a55b-5dc60a73c62c.png)

**Comment by kaixin-hc** [comment]
Oh, I interpreted it as referring to the tasks by proximity, and I expected what needed the attention were the tasks, not the list itself. Especially as the next sentence uses a plural pronoun for what needs to be paid attention to (if the intent was to focus on the list, maybe the plural pronoun shouldn't be used in the next sentence).

**Comment by AllardQuek** [comment]
Yup, makes sense!

**Comment by damithc** [comment]
Thanks all. Will fix in the next version.
